css/styles.css

```diff
@@ -4,39 +4,68 @@
   box-sizing: none;
 }
 
-header,
-footer {
+header {
   color: rgb(92, 145, 145);
 }
 
 body {
-  margin: 28px 40px;
+  margin: 20px 20px;
 }
 
-a[href^="https"]:visited {
+a[href^="https"] {
   text-decoration: none;
-  color: rgb(196, 158, 109);
+  color: burlywood;
 }
 
 a[href^="#"] {
   text-decoration: none;
-  color: rgb(196, 158, 109);
+  color: burlywood;
 }
 
 h1 {
   color: darkslategray;
+  font-size: 40px;
+  padding: 40px 20px;
+  border: none;
+  margin: 20px 0px;
+  background-color: burlywood;
+  width: 100%;
 }
 
+/*h1::before {
+  content: icon;
+} evtl. icon einfügen*/
+
 .menu {
-  border: 3px;
+  width: 100%;
+  display: flex;
+  justify-content: space-around;
+  /* flex-direction: row; zu welchem element?*/
+  flex-wrap: wrap;
+  position: fixed;
+  border: solid;
+  border-width: 2px;
   border-color: rgb(68, 115, 115);
+  border-radius: 4px;
+  background-color: rgb(68, 115, 115);
+  font-size: 20px;
 }
 
+/*ul {
```

Comment:
`.menu {
display: flex;
flex-direction: row;
gap: 10px;
}`

Damit sind die A-Tags die Kind-Elemente und flex-box würde richtig funktionieren. Ich empfehle dir die A-Tags in eine Unordered-List zu packen dieser die Flex-Klasse zuzuweisen und die A-Tags in in die LI-Elemente zu setzen.

Etwa so: 

`<ul class="flex"><li><a href="#about-me">ABOUT ME</a></li>...</ul>`